frontend/src/components/UsernameMenu.tsx

```diff
@@ -1,35 +1,43 @@
-import { useAuth0 } from '@auth0/auth0-react';
+import { CircleUserRound } from 'lucide-react';
 import {
   DropdownMenu,
   DropdownMenuContent,
   DropdownMenuItem,
   DropdownMenuTrigger,
-} from '@radix-ui/react-dropdown-menu';
-import { Separator } from '@radix-ui/react-separator';
-import { CircleUserRound } from 'lucide-react';
+} from './ui/dropdown-menu';
+import { useAuth0 } from '@auth0/auth0-react';
 import { Link } from 'react-router-dom';
+import { Separator } from './ui/separator';
 import { Button } from './ui/button';
 
-const UsernameMenu: React.FC = () => {
+const UsernameMenu = () => {
   const { user, logout } = useAuth0();
 
   return (
     <DropdownMenu>
-      <DropdownMenuTrigger className="flex items-center gap-2 px-3 font-bold hover:text-orange-500">
+      <DropdownMenuTrigger className="flex items-center px-3 font-bold hover:text-orange-500 gap-2">
         <CircleUserRound className="text-orange-500" />
         {user?.email}
       </DropdownMenuTrigger>
       <DropdownMenuContent>
-        <DropdownMenuItem asChild>
+        <DropdownMenuItem>
+          <Link
+            to="/manage-restaurant"
+            className="font-bold hover:text-orange-500"
+          >
+            Manage Restaurant
+          </Link>
+        </DropdownMenuItem>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/28#issuecomment-2385674118>#29443f</a><span> - 10/01/2024, 12:44 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'Manage Restaurant' menu item is using a nested Link component inside a DropdownMenuItem, which may lead to nested interactive elements and potential accessibility issues.
</div>
<div id="fix">
<b>Fix:</b> Restructure the DropdownMenuItem to use the 'asChild' prop and directly wrap the Link component, eliminating the nesting of interactive elements.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,8 +23,8 @@
 -        <DropdownMenuItem>
 -          <Link
 -            to="/manage-restaurant"
 -            className="font-bold hover:text-orange-500"
 -          >
 -            Manage Restaurant
 -          </Link>
 -        </DropdownMenuItem>
 +        <DropdownMenuItem asChild>
 +          <Link
 +            to="/manage-restaurant"
 +            className="font-bold hover:text-orange-500"
 +          >
 +            Manage Restaurant
 +          </Link>
 +        </DropdownMenuItem>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged